### DISCUSSION TITLE
Identical twins USA

### DISCUSSION THREAD

**Discussion by alphonsederus2** [moj-analytical-services/splink#211] [Q&A] [answered]
Often in the US, identical twins share a lot in life besides date of birth. They are born in the same city/state, their social security numbers are often off by 1, and they often have names beginning with the same first letter. Assuming the following:
1. Twins James and John were born on the same date, same address, have the same sex assigned at birth, and social security number is off by 1.
2. James and not John is dfA 
3. John and not James is in dfB

Would we be able to disambiguate or use some type of custom rule to indicate a high chance of twins?

**Comment by RobinL** [comment, marked as answer]
In my experience, the answer is no, but it depends on the data

The problem is that there are two categories of record comparison:
1. Genuine duplicates with some transcription errors (e.g. typo in social security number, a nickname used in one record).
2. Twins

Both of these scenarios occur, and the look 'equivalent' to the computer.

The more information you have to differentiate twins the better.  If you only have first names to distinguish them, I think it's almost impossible.  If you have social security number and middle name, I would expect it probably would be possible, especially if the data quality was high on the social security numbers (implying that a mismatch on SSN should give a strong negative match weight)
